core/state_processor.go

```diff
@@ -589,12 +595,16 @@ func applyTransaction(msg types.Message, parent *types.WorkObject, config *param
 	// Apply the transaction to the current state (included in the env).
 	result, err := ApplyMessage(evm, msg, gp)
 	if err != nil {
-		return nil, err
+		return nil, nil, err
 	}
 
 	var ETXRCount int
 	var ETXPCount int
 	for _, tx := range result.Etxs {
+		// coinbase transactions do not count for the etx limit
```

Comment:
It's impossible for a coinbase tx to emit ETXs

Reply:
deleted